garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -25,15 +25,15 @@ L.round_traitors_more = "Verräter, dies sind die Namen deiner Verbündeten: {na
 L.win_time         = "Die Zeit ist abgelaufen. Die Verräter haben verloren."
 L.win_traitor      = "Die Verräter haben gewonnen!"
 L.win_innocent     = "Die Verräter wurden besiegt!"
-L.win_showreport   = "Schauen wir uns den Runden Report für {num} Sekunde(n) an."
+L.win_showreport   = "Schauen wir uns den Rundenbericht die nächsten {num} Sekunde(n) an."
```

Comment:
Somehow it would be correct, however, there is one problem.
If for example the time is "1" then the sentence would be:
`Schauen wir uns den Rundenbericht die nächsten 1 Sekunde an.`
It might be good to put the "n" in brackets, as I have already done in some other parts of the translation.
Like `nächste(n)`.